packages/db/test/pinning.spec.js

```diff
@@ -403,11 +403,11 @@ describe('Pin Request', () => {
     })
 
     it('does not select pin request after deletion', async () => {
-      assert.rejects(client.getPsaPinRequest(authKey, parseInt(aPinRequestOutput._id, 10)))
```

Comment:
I think we need this `parseInt`? I remember adding it because of type errors.

Reply:
This is now a string and not a bigint serial. Hence this has changed, I've added other comments to make sure we update this across the codebase.

Reply:
yeah that occurred to me after, ignore me :D